src/Tribe/Editor/Compatibility/Classic_Editor.php

```diff
@@ -264,6 +264,11 @@ public function filter_tribe_editor_should_load_blocks( bool $should_load_blocks
 			$should_load_blocks = false;
 		}
 
+		// The "forget" param still overrides all...
```

Comment:
Either full-stops, exclamation or interrogation marks to close a comment line. This comment does not really explain what is happening here and the `forget` param is mentioned but not found near this piece of code. Am I missing something?